reference/7.0/Microsoft.PowerShell.Utility/Get-Date.md

```diff
@@ -507,20 +510,29 @@ The valid **UFormat specifiers** are displayed in the following table:
 | `%R` | Time in 24-hour format -no seconds                                      | 17:45                    |
 | `%r` | Time in 12-hour format                                                  | 09:15:36 AM              |
 | `%S` | Seconds                                                                 | 05                       |
-| `%s` | Seconds elapsed since January 1, 1970 00:00:00                          | 1150451174               |
+| `%s` | Seconds elapsed since January 1, 1970 00:00:00 (UTC)                    | 1150451174               |
 | `%t` | Horizontal tab character                                                |                          |
 | `%T` | Time in 24-hour format                                                  | 17:45:52                 |
 | `%U` | Same as 'W'                                                             |                          |
-| `%u` | Day of the week - number                                                | Sunday = 0               |
+| `%u` | Numeric day of the week (0-6)                                           | Sunday = 0, Saturday = 6 |
 | `%V` | Week of the year                                                        | 01-53                    |
-| `%w` | Same as 'u'                                                             |                          |
+| `%w` | Numeric day of the week (0-6)                                           | Sunday = 0, Saturday = 6 |
 | `%W` | Week of the year                                                        | 00-52                    |
 | `%X` | Same as 'T'                                                             |                          |
 | `%x` | Date in standard format for locale                                      | 06/27/19 for English-US  |
 | `%Y` | Year in 4-digit format                                                  | 2019                     |
 | `%y` | Year in 2-digit format                                                  | 19                       |
 | `%Z` | Time zone offset from Universal Time Coordinate (UTC)                   | -07                      |
 
+> [!NOTE]
+> Window PowerShell's behavior with `Get-Date -UFormat %s` is incorrect in two respects:
+>
+> - The return value is based on local time instead of UTC time.
+> - The string representation of the seconds value has a fractional part. The output is
+>   culture-sensitive with respect to the decimal mark.
+>
+> These behaviors have been fixed in PowerShell 6 and higher.
```

Comment:
Do we still need this note in 7+? If so, should the phrasing be slightly different to call attention to its use for scripts meant to be run on multiple versions and folded in with the alert above?